src/wally-connector.ts

```diff
@@ -436,7 +535,7 @@ class WallyConnector {
       );
     }
 
-    return null;
+    return Promise.reject(`Invalid response for ${method}`);
```

Comment:
I think reject usually passes in an error.
```suggestion
    return Promise.reject(new Error()`Invalid response for ${method}`);
```

Reply:
👍